RELEASE.md

```diff
@@ -0,0 +1,67 @@
+based off of a previously released version as a hotfix
+4. A release candidate will be deployed to preview/preprod by SRE
+5. Community will be notified of release candidate and given a few days to provide feedback
+6. Release Squad Lead will draft Release notes
+7. Head of Engineering will meet with release team and identify if release should be published as stable or beta (pre-release) in GitHub. stable releases can go all the way to mainnet, beta should only go to preprod and wait for that release to be relabeled as stable or a new stable release to be cut.
+8. Release is published
+
+# Rotating Release Engineer Role
+
+All sprints are aligned across the node and it's components. At the end of a sprint cycle the new rotating release engineer is decided on by the leadership team.
+This person's primary duties are integration of new releases of dependencies up the stack to the node. They serve this role until the release is finalized
+according to the above release process (ideally 1 sprint cycle).
+
+## Sub-Teams
+
+### SRE (Site Reliability & Engineering)
+
+The SRE team provides the tooling for monitoring, logging and measurement of live environments. The team initiates deployments of new versions to developer
+testnets, public testnets and production systems. They are responsible for updating dashboards/alerts to align with new node features/refactoring.
+
+### DevX (Developer Experience)
+
+The DevX team is responsible for CI/CD, the building process (using nix and compiling manually), OCI images (e.g. docker containers), systemd services
+and helper scripts associated in running the node for local development and remote deployment purposes.
+
+### Test Engineers
+
+The test engineers are responsible for writing and running integration tests from `cardano-node-tests` repository. They execute integration tests as well as
+tests that measure node synchronization times between releases.
+
+### Performance Engineers
+
+Performance engineers run benchmarks of the node and report any improvements/regressions between node versions.
+
+# Versioning
+
+The node uses a "pseudo semantic versioning" that takes into account breaking change (e.g. new eras) in the versioning logic. This new versioning
+standard within the node is supported as of `8.0.0` release. The first part of the version always references the max protocol version allowed
+on a stable network with no additional experimental override flags. This first part will remain `8` referencing Babbage era until the next era (Conway)
+is finalized. The second digit is a new release incremented counter. Every release based off the master trunk will increment this number by one. The final
+part should always be `0` *unless* an emergeency hard fork is necessary that cannot wait until the next major release. This should be a rare occurrence going
```

Comment:
A controlled hard fork implies a protocol version and consequently the major version bump. So, shouldn't the last version be bumped only for fixes that do not introduce breaking changes?

I would phrase it something like:
```suggestion
part should always be `0` *unless* an emergency bug fix is necessary that cannot wait until the next major release. This should be a rare occurrence going
```

Regardless of the wording it needs a spellfix:
```suggestion
part should always be `0` *unless* an emergency hard fork is necessary that cannot wait until the next major release. This should be a rare occurrence going
```